src/interfaces/pyo3/mod.rs

```diff
@@ -1,9 +1,11 @@
+#[macro_use]
+mod macros;
+mod py_abe_policy;
+mod py_cover_crypt;
+
 use crate::error::Error;
 use pyo3::{pymodule, types::PyModule, PyResult, Python};
 
-pub mod py_abe_policy;
-pub mod py_cover_crypt;
-
 use py_abe_policy::{Attribute, Policy, PolicyAxis};
 use py_cover_crypt::{CoverCrypt, MasterSecretKey, PublicKey, SymmetricKey, UserSecretKey};
```

Comment:
What are these lines used for ?

Reply:
Okay, I didn't see the rest of the file in the diff. I guess I was a bit surprised not to see any public declaration here.